draft-ietf-tls-tls13.md

```diff
@@ -4629,6 +4676,20 @@ resumption master secret computed by connection N and needed to form
 connection N+1 is separate from the traffic keys used by connection N,
 thus providing forward secrecy between the connections.
 
+The PSK binder value forms a binding between a PSK
+and the current handshake, as well as between the session where the
+PSK was established and the session where it was used. This binding
+transitively includes the original handshake transcript, because that
+transcript is digested into the values which produce the Resumption
+Master Secret. This requires that both the KDF used to produce the RMS
+and the MAC used to compute the binder be collision
+resistant. These are properties of HKDF and HMAC respectively when
+used with collision resistant hash functions and producing output of
+at least 256 bits.  Any future replacement of these functions MUST
```

Comment:
Wasn't there a comment about this being the hash output length and not necessarily 256 bits?  I forget how that got resolved.